cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -666,6 +666,27 @@ export const connectorDetails = {
         },
       },
     }),
+    VoidAfterConfirm: getCustomExchange({
+      Request: {},
+      Response: {
+        status: 200,
+        body: {
+          status: "cancelled",
+          capture_method: "manual",
+        },
+      },
+      ResponseCustom: {
+        status: 400,
+        body: {
+          error: {
+            type: "invalid_request",
+            message:
+              "You cannot cancel this payment because it has status succeeded",
+            code: "IR_16",
+          },
+        },
+      },
+    }),
```

Comment:
again, this will affect all the connectors since you've modified `PaymentTest`

Reply:
This is handled by fallbackData which is fetched from `Commons.js`